### PR TITLE
add loading conditional to action battle detail

### PR DESCRIPTION
**Pull request by DeborahAlmeida**
## Objective and Motivation
- Improvement of the actions to handle loading status

### CODE REVIEW

File: frontend/js/pages/BattleDetail.js [hunk 1]
```diff
@@ -10,6 +10,8 @@ import { orderTeamsByCurrentUser } from '../utils/battle-detail';
 import Urls from '../utils/urls';
 
 function BattleDetail(props) {
+  const { loading } = props.battle;
+  const { error } = props.battle;
   const { id } = useParams();
   const { battle } = props.battle;
   const { user } = props.user;
```
Comment (rsarai) [inline frontend/js/pages/BattleDetail.js]: All props can be deconstructed in a single line

File: frontend/js/pages/BattleDetail.js [hunk 2]
```diff
@@ -21,6 +23,18 @@ function BattleDetail(props) {
...
+        src="https://media0.giphy.com/media/3oEjI6SIIHBdRxXI40/200.gif"
+      />
+    );
+  }
+  if (error) {
+    return 'Ocurred an error';
+  }
   if (!battle) {
     return '';
   }
```
Comment (rsarai) [inline frontend/js/pages/BattleDetail.js]: This loading states can be better handled don't you think?

**Review by rsarai — CHANGES_REQUESTED** [review]

File: frontend/js/pages/BattleDetail.js (hunk 1 above)
Comment (DeborahAlmeida) [reply frontend/js/pages/BattleDetail.js]: Thanks! I didn't realize :)

File: frontend/js/pages/BattleDetail.js (hunk 2 above)
Comment (DeborahAlmeida) [reply frontend/js/pages/BattleDetail.js]: Hmm great! I'll improve it!

**Review by rsarai — APPROVED** [review]

**Pull request closed without merging**